src/box/box.h

```diff
@@ -620,6 +658,27 @@ box_iproto_send(uint64_t sid,
 		const char *header, const char *header_end,
 		const char *body, const char *body_end);
 
+/**
+ * Sets an IPROTO request handler with the provided context for the
+ * given request type.
+ *
+ * NB: yields.
+ *
+ * \param req_type request type code from iproto_type enumeration
+ *                 (src/box/iproto_constants.h);
+ *                 use IPROTO_UNKNOWN for overriding the handler of unknown
+ *                 request types;
+ * \param handler IPROTO request handler; passing NULL resets the corresponding
+ *                handler
+ * \param destroy IPROTO request handler destructor
+ * \param ctx context passed to handler
+ * \retval -1 on error (check box_error_last())
+ * \retval 0 on success
+ */
+API_EXPORT int
+box_iproto_override(uint32_t req_type, iproto_handler_t handler,
+		    iproto_handler_destroy_t destroy, void *ctx);
```

Comment:
@CuriousGeorgiy Please update the design document.

Reply:
Done.